mm2src/coins/nft/nft_structs.rs

```diff
@@ -717,7 +726,9 @@ pub(crate) struct NftCtx {
     #[cfg(target_arch = "wasm32")]
     pub(crate) nft_cache_db: SharedDb<NftCacheIDB>,
     #[cfg(not(target_arch = "wasm32"))]
-    pub(crate) nft_cache_db: Arc<AsyncMutex<AsyncConnection>>,
+    pub(crate) nft_cache_dbs: AsyncSqliteConnPool,
+    #[cfg(not(target_arch = "wasm32"))]
+    ctx: MmArc,
```

Comment:
This is going to cause a self referencing problem I think. If you want to access ctx from nft context, add `MmWeak` instead (considering there is no any other way possible).

Reply:
removed and cleanup sql connection pool a bit